pages/gm/f21.js

```diff
@@ -84,16 +111,86 @@ function F21() {
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/icpc_wordmark.svg" alt="icpc wordmark"/> Jacob Zhang</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/design_wordmark.svg" alt="design wordmark"/> Crystal Huynh, Muthu Palaniappan</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/cyber_wordmark.svg" alt="cyber wordmark"/> Josh Lee, Henry Chang</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/teachLA_wordmark.svg" alt="teachLA wordmark"/> Sophie Schoenmeyer, Chloe Uy</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/w_wordmark.svg" alt="w wordmark"/> Cindy Zhang, Lena O&apos;Grady</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/ai_wordmark.svg" alt="ai wordmark"/> Justin Yi</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/hack_wordmark.svg" alt="hack wordmark"/> Asha Kar, Eugene Lo</p>
+					</div>
+          <div className="gm-program-section">
+            <div className="section-header">
+              <h3>ACM Board</h3>
+              <h4>How to become even more involved with ACM beyond attending workshops and events</h4>
+            </div>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/board_wordmark.svg" alt="board wordmark"/>&nbsp;&nbsp;External: Tina Huang</p>
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <p className="gm-program-row"><img src="/images/committees/board_wordmark.svg" alt="board wordmark"/>&nbsp;&nbsp;Internal: Evan Zhong</p>
+          </div>
+          <div className="gm-program-section">
+            <div className="section-header">
+              <h3>ACM Initatives</h3>
+              <h4>See the exciting new things ACM is trying out</h4>
+            </div>
+            <p className="gm-program-row">ACM Impact: Maya Raman</p>
+            <p className="gm-program-row">ACM JEDI: Helia Woo</p>
+          </div>
+          <div className="gm-program-section">
+            <div className="section-header">
+              <h3>Tabling and Social</h3>
+              <h4>Interact with ACM's officers and maybe walk away with new friends!</h4>
+            </div>
+            <p className="gm-program-row">Christina Tong, Rish Jain</p>
```

Comment:
Is this the correct group of people? I would imagine something like "All ACM Officers" might make more sense, though I could be completely wrong.

Reply:
They're introducing it, but I can make the change